talkmeup/templates/home.html

```diff
@@ -2,83 +2,195 @@
 
 {% block content %}
 
-  <div class="embed-responsive embed-responsive-16by9">
-    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/Tiy2LONr050?autoplay=1&loop=1&playlist=Tiy2LONr050&mute=1"></iframe>
-  </div>
+<div style="text-align: center">
+<!--  <div class="embed-responsive embed-responsive-16by9">-->
+    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/Tiy2LONr050?autoplay=1&loop=1&playlist=Tiy2LONr050&mute=1" height="550" width="970" ></iframe>
+</div>
+<!--  </div>-->
 
-  <hr class="featurette-divider">
 
-  <div class="container topics">
+<hr class="featurette-divider">
 
+  <div class="container topics" > 
+      
     <div>
       <h1>Professors from top business schools offer 1:1 soft skills training on various topics! Exclusively available at TalkMeUp!</h1>
-      <p><a class="btn btn-primary" href="{% url 'coach:index' %}" role="button">See our coaches</a></p>
+      
+    <p><a class="btn btn-primary" href="{% url 'coach:index' %}" role="button">See our coaches</a></p>
     </div>
 
-    <hr>
-    
-    <div class="topic">
-      <h2>Lead with wisdom and charm</h2>
-      <p><a class="btn btn-primary" href="#" role="button">Express Interest</a></p>
+<hr>
+      
+    <div class="w3-row-padding w3-center">
+        <div class="w3-quarter">
+        <div class="topic">
+            <h2>Elevator pitch</h2>
+            <p>
+                <img src="../../media/categoryImg/professor1.jpg" style="width:100%">
```

Comment:
media is for user uploaded stuffs. These files are static, could you please move the files to the 'static' directory? maybe create a subdir 'images' under 'static', and then you can access the file with something like "{% static 'images/professor1.jpg' %}"